buildSrc/src/main/groovy/stairway.java-conventions.gradle

```diff
@@ -16,8 +16,7 @@ dependencies {
     implementation group: 'org.slf4j', name: 'slf4j-api', version: '1.7.36'
 
     // For testing
-    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: '5.9.0'
-    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: '5.9.0'
+    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter', version: '5.10.2'
```

Comment:
Needed this expanded dependency for parameterized tests, and upgraded while in the neighborhood.